extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -66,4 +66,13 @@ interface TokenCache {
          */
         Optional<Duration> cleanUpTimerInterval();
     }
+
+    static io.quarkus.oidc.runtime.OidcTenantConfig getDefaultTenant(OidcConfig config) {
+        for (var tenant : config.namedTenants().entrySet()) {
+            if (OidcConfig.DEFAULT_TENANT_KEY.equals(tenant.getKey())) {
+                return tenant.getValue();
+            }
+        }
+        return null;
```

Comment:
What is your motivation to write this loop rather then do `config.namedTenants().get(OidcConfig.DEFAULT_TENANT_KEY);`

Reply:
Ah, I see it, you used `@WithDefaults`. I withdraw this question.